met-api/migrations/versions/e5d3bbb8d2f0_add_survey_hidden_column.py

```diff
@@ -0,0 +1,28 @@
+"""add_survey_hidden_column
+
+Revision ID: e5d3bbb8d2f0
+Revises: 7d8897c412de
+Create Date: 2023-04-05 16:47:19.185256
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'e5d3bbb8d2f0'
+down_revision = '7d8897c412de'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('survey', sa.Column('is_hidden', sa.Boolean(), nullable=True))
```

Comment:
I suggest to do something like this. Add the column as nullabe. Update all the null values to False then make it nullable False and default to False
![image](https://user-images.githubusercontent.com/91914654/230437369-e1c90eeb-9196-4740-8bba-583c34d317e1.png)